### ISSUE TITLE
Please make permission users group for project

### ISSUE THREAD

**Issue by hasket** [reportportal/reportportal#1144]
Hello, I work in a large company with a large group of testers and a large number of projects in the reportportal.
In version 5.3, I encountered the problem of issuing the rights of a specific person to a group of projects. There are a lot of people, there are many projects, then you have to poke a lot on a person and give him access to the project.
I propose to make at least a group for users so that there are fewer actions for granting rights to projects, thanks.

**Comment by evjlobanova** [comment]
Do you mean to have a possibility to make a bulk assign operation or something else?

**Comment by hasket** [comment]
Yes, it is possible as an option, but people came up with a long time ago that it is easier to combine several users into groups for further assigning a group to a project.
thanks
-------------------------------------------------
На русском,
Да, объединение в группы пользователей необходимы для дальнейшего назначения группы на конкретные проекты массово, это повысит юзабилити и удобство работы с приложением, спасибо.
это очень сильно упростит работу с приложением.
